maths/bisection.py

```diff
@@ -0,0 +1,49 @@
+"""Given a function f(x) on floating number x and two numbers ‘a’ and ‘b’
+such that f(a)*f(b) < 0 and f(x) is continuous in [a, b]. Here f(x) represents algebraic or transcendental equation.
+Find root of function in interval [a, b] (Or find a value of x such that f(x) is 0)
+"""
+import doctest
+
+
+def equation(x: int) -> int:
+    return 10 - x * x
```

Comment:
Please add doctests here for x = 5, 0, and -5

Reply:
negative results have a problem with doctests